spring-integration-core/src/main/java/org/springframework/integration/dsl/QueueChannelSpec.java

```diff
@@ -31,9 +33,9 @@
  */
 public class QueueChannelSpec extends MessageChannelSpec<QueueChannelSpec, QueueChannel> {
 
-	protected Queue<Message<?>> queue; // NOSONAR
+	protected @Nullable Queue<Message<?>> queue; // NOSONAR
```

Comment:
Curious.  Why do we need to keep the `//NOSONAR`?

Reply:
Because this property is set from extension classes.
Yes, we may remove it since does not look like anything currently in our CI/CD complains 😄